tests/testComposite.py

```diff
@@ -50,41 +65,47 @@ def tearDown(self):
         if os.path.exists(self.output):
             shutil.rmtree(self.output)
 
-    def test(self):
+    def testGet(self):
         """Verify that a composite can be loaded and that its components are the same as when the type1
         components are loaded individually (verifies correct lookup in this case).
+
         Also verify that when the individual components are put and when the composite is put (which
```

Comment:
I don't think this docstring applies any more, since this test is only testing get.